airflow/bin/cli.py

```diff
@@ -1516,8 +1519,9 @@ def _import_users(users_list):
         roles = []
         for rolename in user['roles']:
             role = appbuilder.sm.find_role(rolename)
+            valid_roles = appbuilder.sm.get_all_roles()
```

Comment:
Can you move this inside of the `if` statement?

Reply:
Done